nixos/modules/services/web-apps/stash.nix

```
@@ -0,0 +1,95 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.stash;
+in
+{
+  options = {
+    services.stash = {
+      enable = mkEnableOption "An organizer for your porn, written in Go.";
```

```suggestion
      enable = mkEnableOption "an organizer for your porn, written in Go";
```

This will become "Whether to enable an organizer..."

https://github.com/NixOS/nixpkgs/blob/b428e0002d0020df87eedd6b9716870630fbba5d/lib/options.nix#L98-L105